include/regs.h

```diff
@@ -58,15 +58,26 @@
 #define R_FB_FILTER_ENV_COLOR(c)                 SREG(85 + (c))
 #define R_ENABLE_FB_FILTER                       SREG(88)
 #define R_PAUSE_MENU_MODE                        SREG(94)
+#define R_CAM_XZ_UPDATE_RATE                     OREG(2)
+#define R_CAM_Y_OFFSET_UPDATE_RATE               OREG(3)
+#define R_CAM_FOV_UPDATE_RATE                    OREG(4)
 #define R_CAM_MAX_PITCH                          OREG(5)
+#define R_CAM_R_UPDATE_RATE_INV                  OREG(6)
 #define R_CAM_DEFAULT_PITCH_UPDATE_RATE_INV      OREG(7)
```

Comment:
`R_CAM_DEFAULT_PITCH_UPDATE_RATE_INV` currently has `DEFAULT` in it's UPDATE_RATE macro, I'd say add it too all or remove it from the pitch update. I'm inclined to remove the `DEFAULT` since these are verbose enough as is